test/verify/files/6050.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
> This file needs a script, or documentation on how to create it.

I will change the test to download them from the candlepin machine directly, as part of the test.